scar-cpu.yml

```diff
@@ -4,6 +4,7 @@ channels:
 - pytorch
 - conda-forge
 dependencies:
+- setuptools<=59.5.0
```

Comment:
Here try to remove it again and see if it is still working...
If this is required then add it in the scar-gpu.yml as well